--- isapnptools-1.26/configure.in~	Wed Apr  9 09:03:43 2003
+++ isapnptools-1.26/configure.in	Wed Apr  9 09:04:03 2003
@@ -40,7 +40,7 @@
 AC_CHECK_FUNCS(iopl ioperm usleep delay _sleep2)
 
 dnl check for getopt in standard library
-AC_CHECK_FUNCS(getopt_long , , [AC_LIBOBJ([getopt.o, getopt1.o])] ) 
+#AC_CHECK_FUNCS(getopt_long , , [AC_LIBOBJ([getopt.o, getopt1.o])] ) 
 
 AC_ARG_ENABLE(
 	realtime,
