diff -Nru isapnptools-1.18/Makefile isapnptools-1.18.new/Makefile
--- isapnptools-1.18/Makefile	Wed Jun  9 15:23:18 1999
+++ isapnptools-1.18.new/Makefile	Wed Jun  9 15:25:03 1999
@@ -52,22 +52,22 @@
 all:	pnpdump isapnp
 
 install: pnpdump isapnp install.bin install.man
-	mkdir -p $(INSTALLSCRIPTDIR)
-	install config-scripts/* $(INSTALLSCRIPTDIR)/
-	if test ! -f $(CONFDIR)/$(GONEFILE);\
+	mkdir -p $(DESTDIR)$(INSTALLSCRIPTDIR)
+	install config-scripts/* $(DESTDIR)$(INSTALLSCRIPTDIR)/
+	if test ! -f $(DESTDIR)$(CONFDIR)/$(GONEFILE);\
 	then\
-		install $(GONEFILE) $(CONFDIR);\
+		install $(GONEFILE) $(DESTDIR)$(CONFDIR);\
 		echo -e "\nSample preallocated resource file copied to $(CONFDIR)/$(GONEFILE), edit to suit\n";\
 	fi
 
 install.bin: 
-	install -s -m 0755 pnpdump $(INSTALLBINDIR)
-	install -s -m 0755 isapnp $(INSTALLBINDIR)
+	install -s -m 0755 pnpdump $(DESTDIR)$(INSTALLBINDIR)
+	install -s -m 0755 isapnp $(DESTDIR)$(INSTALLBINDIR)
 
 install.man:
-	install -m 0644 isapnp.8 $(INSTALLMANDIR)/man8
-	install -m 0644 isapnp.conf.5 $(INSTALLMANDIR)/man5
-	install -m 0644 pnpdump.8 $(INSTALLMANDIR)/man8
+	install -m 0644 isapnp.8 $(DESTDIR)$(INSTALLMANDIR)/man8
+	install -m 0644 isapnp.conf.5 $(DESTDIR)$(INSTALLMANDIR)/man5
+	install -m 0644 pnpdump.8 $(DESTDIR)$(INSTALLMANDIR)/man8
 
 TAGS: pnpdump.c resource.c pnp-select.c pnp.h isapnp.y
 	etags $^
